src/Filters/remez_fir.jl

```diff
@@ -324,7 +324,7 @@ Construct a length 35 filter with a passband at 0.15-0.4 Hz
 those bands - the transition bands - is unspecified.
 
 ```julia-repl
-julia> bpass = remez(35, [(0, 0.1)=>0, (0.15, 0.4)=>1, (0.45, 0.5)=>0])
```

Comment:
I think it's common to have the `julia>` prompt inside `julia-repl` code blocks. E.g. see the second block [here](https://docs.julialang.org/en/v1/manual/documentation/index.html).